imports/plugins/core/accounts/server/no-meteor/getShopIdForContext.js

```diff
@@ -12,8 +12,15 @@ export default async function getShopIdForContext(context) {
 
   // if still not found, look up the shop by domain
   if (!shopId) {
-    shopId = await getShopIdByDomain(collections);
+    shopId = await getShopIdByDomain(context);
   }
 
+  // if (shopId) {
```

Comment:
would love opinions here. seems odd to _set_  data in a get function, but also makes sense to do it here